app.js

```diff
@@ -32,13 +32,15 @@ app.use('/', express.static(__dirname + '/dist'));
 /** API ENDPOINTS */
 // import the API controllers
 var sampleApi = require('./controllers/sampleController');
+var deviceApi = require('./controllers/deviceController');
```

Comment:
I think in the future, we're gonna name our API plural ("devicesApi", "devicesController"). Just so that it's consistent with the routing you have in line 38